src/pulse3D/excel_writer.py

```diff
@@ -19,6 +19,7 @@
 
 from .constants import *
 from .exceptions import *
+from .metrics import *
```

Comment:
I think we should avoid importing like this, especially if we only need a few things from the module